python/cucim/src/cucim/skimage/restoration/j_invariant.py

```diff
@@ -273,7 +283,7 @@ def calibrate_denoiser(
     ...                                          calibrate_denoiser)
     >>> img = color.rgb2gray(cp.array(data.astronaut()[:50, :50]))
     >>> noisy = img + 0.5 * img.std() * cp.random.randn(*img.shape)
-    >>> parameters = {'weight': cp.arange(0.01, 0.5, 0.05)}
+    >>> parameters = {'weight': cp.arange(0.01, 0.3, 0.02)}
```

Comment:
Am curious about what motivated this change

Reply:
The reason I had a different value was that originally scikit-image was using a different wavelet-based denoiser. Since we didn't have `denoise_wavelet` implemented in cuCIM I switched to `denoise_tv_chambolle` in our docstring and just chose a value that worked okay.

Later, scikit-image made PyWavelets optional and changed their example to use `denoise_tv_chambolle` but happened to choose a different range than what I originally had in cuCIM. I changed to the scikit-image values for consistency, but it doesn't really matter either way.

Reply:
Ah ok. That context was all I was looking for. Thanks Greg! 🙏